src/mono/wasm/build/WasmApp.Native.targets

```diff
@@ -29,7 +29,10 @@
   </ItemGroup>
 
   <ItemGroup Condition="'$(Configuration)' == 'Debug' and '@(_MonoComponent->Count())' == 0">
-    <_MonoComponent Include="hot_reload;debugger" />
+    <_MonoComponent Include="hot_reload;debugger;marshal-ilgen" />
+  </ItemGroup>
+  <ItemGroup Condition="'$(Configuration)' == 'Release' and '@(_MonoComponent->Count())' == 0">
+    <_MonoComponent Include="marshal-ilgen" />
```

Comment:
Just curious, why is this needed by default for Release?

Reply:
Right now, we always need the component (in both Debug and Release). The tooling to drop it when not needed has not been implemented yet, its part three of this issue: https://github.com/dotnet/runtime/issues/61685. 

I guess I could add a branch to here to *always* add it, regardless of Release/Debug, if that would make it clearer.

Reply:
> I guess I could add a branch to here to always add it, regardless of Release/Debug, if that would make it clearer.

I think that would be good.

Reply:
Looking at it again, I am not sure if it is that easy. What is this `@(_MonoComponent->Count())' == 0` condition that was originally there is for? Presumably we only want to add items to _MonoComponent *once* for some reason, but why?

Reply:
I think we are setting those as the default, so if nobody else has added `_MonoComponent`, then we use these defaults. @lambdageek can answer it best